CryptoPkg/Driver/Packaging/generate_cryptodriver.py

```diff
@@ -143,7 +143,7 @@ def get_flavors():
             "guid": "d9a75606-caba-4aa0-80a6-591852335400"

```

Comment:
Do you need to touch "TINY_SHA" to include "SHA512"?

Reply:
Also, do you know if we still have any use cases for "SM3"?

Reply:
We don't have any use cases for SM3 for any current platform.